servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -522,6 +524,36 @@ void convertToLowerCase(Object obj) {
                     domainMeta.setSignAlgorithm(domainMeta.getSignAlgorithm().toLowerCase());
                 }
             }
+        },
+        ASSERTION_CONDITIONS {
+            @Override
+            void convertToLowerCase(Object obj) {
+                AssertionConditions assertionConditions = (AssertionConditions) obj;
+                // lower case key and value, assertion condition operator is an ENUM, hence left unchanged
+                assertionConditions.setConditionsList(assertionConditions.getConditionsList().stream()
+                        .map(c -> new AssertionCondition()
```

Comment:
why are we creating new assertion conditions rather than just updating the values in place like for all objects?